### PR TITLE
Migrate to the new Heroku pg:backups interface

### PR DESCRIPTION
**Pull request by SupremumLimit**
I've updated the script to use the new CLI interface as the old interface is going away.

### CODE REVIEW

File: bin/backup.sh [hunk 1]
```diff
@@ -26,9 +26,10 @@ chmod +x ./awscli-bundle/install
 
 BACKUP_FILE_NAME="$(date +"%Y-%m-%d-%H-%M")-$APP-$DATABASE.dump"
 
-/app/vendor/heroku-toolbelt/bin/heroku pgbackups:capture $DATABASE -e --app $APP
-curl -o $BACKUP_FILE_NAME `/app/vendor/heroku-toolbelt/bin/heroku pgbackups:url --app $APP`
-gzip $BACKUP_FILE_NAME
-/tmp/aws/bin/aws s3 cp $BACKUP_FILE_NAME.gz s3://$S3_BUCKET_PATH/$APP/$DATABASE/$BACKUP_FILE_NAME.gz
+/app/vendor/heroku-toolbelt/bin/heroku pg:backups capture $DATABASE --app $APP
+curl -o $BACKUP_FILE_NAME `/app/vendor/heroku-toolbelt/bin/heroku pg:backups public-url --app $APP`
+#gzip $BACKUP_FILE_NAME
```
Comment (kbaum) [inline bin/backup.sh]: No more gzip needed?

**Comment by kbaum** [comment]
I think this pull request handles the issue:

https://github.com/kbaum/heroku-database-backups/pull/4

**Comment by SupremumLimit** [comment]
Sorry, I think I misunderstood how pull requests work. I was trying to submit a particular commit which still gzips the backup. However, my subsequent commits (for my own purposes) were included as well for some reason.

**Comment by kbaum** [comment]
No problem!

**Pull request closed without merging**